README.md

```diff
@@ -2,20 +2,34 @@
 
 This repository contains code for studying the adversarial robustness of KataGo.
 
-Links:
-* [Paper on arXiv](https://arxiv.org/abs/2211.00241)
-* [Website](https://goattack.alignmentfund.org/)
+Read about our research here: https://arxiv.org/abs/2211.00241.
+
+View our website here: https://goattack.far.ai/.
+
+To run our adversary with Sabaki, see [this guide](sabaki/README.md).
+
+# Development / testing information
+
+To clone this repository,
+run one of the following commands
+```
+# Via https
+git clone --recurse-submodules https://github.com/AlignmentResearch/go_attack.git 
+
+# Via ssh
+git clone --recurse-submodules git@github.com:AlignmentResearch/go_attack.git
+```
 
 ## Git submodule: KataGo-custom
 
-Modifications to KataGo *are not* tracked in this repository and should instead be made to the [HumanCompatibleAI/KataGo-custom](https://github.com/HumanCompatibleAI/KataGo-custom) repository. We use code from KataGo-custom in this repository via Git submodules.
+Modifications to KataGo *are not* tracked in this repository and should instead be made to the [AlignmentResearch/KataGo-custom](https://github.com/AlignmentResearch/KataGo-custom) repository. We use code from KataGo-custom in this repository via Git submodules.
 
 KataGo-custom has the following significant branches:
 
 - `KataGo-custom/stable` contains our changes to the stable version of KataGo.
 - `KataGo-custom/master` tracks https://github.com/lightvector/KataGo.
 
-# Development / testing
+## Other setup
```

Comment:
maybe: remove this section header and move the contents of this section above `##Git submodule: KataGo-custom`

Reply:
Moved higher.